msteams-platform/graph-api/meeting-transcripts/api-transcripts.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Use Graph APIs to fetch transcript
+description: Describes the APIs to fetch meeting transcripts
+ms.localizationpriority: high
+ms.topic: concept
+---
+# Use Graph APIs to fetch transcript
```

Comment:
This page will need to be updated once the API reference docs that @mankadnandan is working on go live. The idea is that we shouldn't duplicate API docs in two places. It's really hard to maintain going forward.